aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -539,6 +563,7 @@ public void setGlueCatalogId(String id) {
   public boolean glueCatalogSkipArchive() {
     return glueCatalogSkipArchive;
   }
+
```

Comment:
Nit: Unnecessary new line

Reply:
It's done by IntelliJ formatter. I think the previous commit forgot this one.
 
![image](https://user-images.githubusercontent.com/6848311/179394546-c87760e7-7fa4-42a5-81b4-98b4853c43ec.png)